Linux_scripts/RHUI_repo_validation_scripts/repo_check.py

```diff
@@ -131,6 +150,38 @@ def error_check():
          msg_new_scenario()
    error_check() 
 
+
+def disclaimer():
+   logging.info("\n\n Disclaimer : This script helps in detecting RHUI server connectivity issues only for Redhat-6,7 and 8 Pay as you go images deployed from Azure market place.Upon execution this script does not modify anything on the VM instead it will provide suggestion for fix based on the error. Always, Please download the latest version of the script from git hub page which will help in effective troubleshooting and also have bugs addressed.\n")
```

Comment:
just check and correct the disclaimer message: eg: place.Upon -> place. Upon
Always, Please -> Always
git hub -> GitHub page (add link too)

Reply:
This is Done.